imap/mailbox.c

```diff
@@ -4799,8 +4815,12 @@ EXPORTED int mailbox_append_index_record(struct mailbox *mailbox,
         }
     }
 
-    if (!record->internaldate.tv_sec)
-        record->internaldate.tv_sec = time(NULL);
+    clock_gettime(CLOCK_REALTIME, &now);
```

Comment:
This clock_gettime() could be moved into the if block I think? Doesn't seem like now is used anywhere else

Reply:
Yep.  Will do.  I think I did this to keep the diff smaller, but that was dumb.